static/app/views/settings/organizationDeveloperSettings/sentryApplicationRow/sentryApplicationRowButtons.spec.tsx

```diff
@@ -0,0 +1,174 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {SentryAppFixture} from 'sentry-fixture/sentryApp';
+
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import SentryApplicationRowButtons from 'sentry/views/settings/organizationDeveloperSettings/sentryApplicationRow/sentryApplicationRowButtons';
+
+describe('Sentry App Row Buttons', function () {
+  const removeApp = jest.fn();
+  const publishApp = jest.fn();
+  const sentryApp = SentryAppFixture();
+
+  afterEach(() => {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('renders', async () => {
+    render(
+      <SentryApplicationRowButtons
+        organization={OrganizationFixture()}
+        app={sentryApp}
+        onClickRemove={removeApp}
+        onClickPublish={publishApp}
+      />
+    );
+
+    const publishButton = await screen.findByRole('button', {name: 'Publish'});
+    expect(publishButton).toBeEnabled();
+    await userEvent.hover(publishButton);
```

Comment:
if you want to check the hover tooltip message you could do something like this

https://github.com/getsentry/sentry/blob/2fbf8abdf8d9c5498fe75d3d9c692cab39fe5b20/static/app/components/workflowEngine/gridCell/index.spec.tsx#L75-L78

Reply:
this worked!!!!!!!! thank you 🙇‍♀️